### PR TITLE
attach page status popin at bottom right

### PR DESCRIPTION
**Pull request by ndufreche**
Page status popin is attach to bottom right and stay at this place on navigator re-size.

### CODE REVIEW

File: src/tb/apps/page/widgets/InformationPage.js [hunk 1]
```diff
@@ -37,9 +38,33 @@ define(
...
+                        return (window.innerHeight - 200);
+                    },
+                    resizeWidth = function () {
+                        return (window.innerWidth - 200);
+                    };
+
+                this.popin = PopinManager.createPopIn({
+                    open: function () {
+                        parent = jQuery(this).parent('.ui-dialog:first');
+                        parent.css({
```
Comment (mickaelandrieu) [inline src/tb/apps/page/widgets/InformationPage.js]: some duplicate, do you often have this kind of needs ?

**Comment by fkroockmann** [comment]
Use http://api.jqueryui.com/dialog/#option-position instead of custom positionning

**Comment by mickaelandrieu** [comment]
@ndufreche is this pull request ready to be merged ?

**Comment by ndufreche** [comment]
For me, I have tried the @fkroockmann's solution, but without success.

**Pull request merged**